### PR TITLE
Add AppVeyor config for Windows builds

### PR DESCRIPTION
**Pull request by isaachier**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2014, Ruslan Baratov
```
Comment (isaachier) [inline appveyor.yml]: Not sure how to handle license. Took this from my CMake mentor @ruslo's repo here: https://github.com/forexample/appveyor-test/blob/master/appveyor.yml.

**Comment by dfellis** [comment]
I wholeheartedly approve of this change, but will have to leave everything to @isaacbrodsky . The copyright notice in the file might be an issue for legal that he'll have to find out, and he'll have to get the appveyor account set up for Uber's repos and integrate it into H3 to confirm it can run tests in this branch correctly.

**Comment by isaachier** [comment]
Just for reference, fixes the Windows part of #34.

**Comment by isaachier** [comment]
sure

**Pull request merged**